.example.env

```diff
@@ -1,8 +1,13 @@
 SECRET_SESSION_PASSCODE=
 CLIENT_ID=
 CLIENT_SECRET=
+CLIENT_TOKEN=
 REDIRECT_URL=http://localhost:3000/api/auth/redirect
+INVITE_REDIRECT_URL=http://localhost:3000/api/auth/invite_redirect
 ENCRIPT_KEY=
```

Comment:
Was this meant to be encript or encrypt?

Reply:
That is a typo yes, Thanks for catching it!

As well as `GUILD_ENCRIPT_KEY` but that will be removed.

Reply:
I was just skimming through the code, good to know that I can contribute to something that I have nothing to do with. Looking forward to the release :)